srcpkgs/gmsl/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'gmsl'
+pkgname=gmsl
+version=1.1.8
+revision=1
+noarch=yes
+create_wrksrc=true
+depends="make"
+short_desc="GNU Make Standard Library"
+maintainer="B. Wilson <x@wilsonb.com>"
+license="BSD-3-Clause-Clear"
```

Comment:
please don't vendor the license

Reply:
Sorry, I don't understand what "vendor the license" means.

Reply:
the license seems to be regular BSD-3-Clause and can be extracted with sed from the `gmsl` file